lib/jira/core.rb

```diff
@@ -61,17 +60,10 @@ def ticket?(ticket, verbose=true)
       ### Relevant Paths
 
       #
-      # @return [String] path to .jira-url file
+      # @return [String] path to .jira-cli file
       #
-      def url_path
-        @url_path ||= self.root_path + "/.jira-url"
-      end
-
-      #
-      # @return [String] path to .jira-auth file
-      #
-      def auth_path
-        @auth_path ||= self.root_path + "/.jira-auth"
+      def cli_path
+        @cli_path ||= self.root_path + "/.jira-cli"
```

Comment:
The project hasn't had any changes because it is probably complete. Gem is pretty stable by now I would think. https://rubygems.org/gems/inifile/versions/3.0.0